src/main/java/ttt/HumanPlayer.java

```diff
@@ -23,7 +24,8 @@ public int chooseNextMoveFrom(Board board) {
         return asInteger(usersInput);
     }
 
-    public PlayerSymbol getSymbols() {
+    @Override
+    public PlayerSymbol getSymbol() {
```

Comment:
Guessing there's duplication amongst the different Player implementations, might want to consider abstract Player class instead of an interface. I don't usually recommend preferring inheritance...